audio_helper/src/main.c

```diff
@@ -0,0 +1,125 @@
+//
+// Copyright (c) 2015 Assured Information Security, Inc
+//
+// Dates Modified:
+//  - 4/8/2015: Initial commit
```

Comment:
My personal opinion is that this type of changelog header is not useful (in this case)... your git history will take care of this.